act-rules-format.md

```diff
@@ -170,16 +170,18 @@ Each [=accessibility requirement=] in the mapping <em class="rfc2119">must</em>
 4. the conformance level associated with the accessibility requirement, if one exists.
 
 
-### Outcome Mapping ### {#outcome-mapping}
+### Mapping Conformance Requirements ### {#mapping-conformance-requirements}
 
-For each accessibility requirement in the mapping, an ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying that accessibility requirement for that [=test subject=]. When one or more of the outcomes for a test target is `failed`, the accessibility requirements are <dfn>not satisfied</dfn> for the test subject. When all of the outcomes are `passed` or `inapplicable`, the accessibility requirements could be <dfn>satisfied</dfn>, or <dfn>further testing is needed</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
+An ACT Rule <em class="rfc2119">must</em> indicate what the [=outcomes=] of the rule mean for satisfying an accessibility requirement for that [=test subject=]. A conformance requirement mapped in a rule is an accessibility requirement that is <dfn>not satisfied</dfn> when all of the outcomes for a test target in failed test cases is `failed`. When all of the outcomes for passed test cases or inapplicable test cases are `passed` or `inapplicable`, respectively, the mapped conformance requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>. Rules that can be used to determine if an accessibility requirement is *satisfied* are called <dfn>satisfying tests</dfn>.
```

Comment:
Looking at this harder, I'm a bit confused by the wording here. In our definition of outcome (https://www.w3.org/TR/act-rules-format/#outcome) we specify in a note that there is:

> one passed or failed outcome for every [test target](https://www.w3.org/TR/act-rules-format/#test-target)

I'm actually not quite sure what we mean when we reference multiple outcomes for a single test target, is it:
1. An outcome for each of the procedures that a `tester` is using.
2. An outcome for each test target in each failed test case. 
3. An outcome for each accessibility requirement being mapped.
4. Confusing the test target with the test subject in some way.
5. Something else?

Reply:
Kind of an off topic note, but is the last sentence still reasonable to include? I haven't heard of us use the `satisfying tests` definition in practice ever, I had forgotten it was in the spec.

Reply:
Following our conversation this morning, I think the wording I might be leading towards is something along the lines of:

> A conformance requirement mapped in a rule is an accessibility requirement that is <dfn>not satisfied</dfn> when the outcome for any test target within the test subject is failed. When all of the outcomes for the test targets within the test subject are passed or inapplicable, the conformance requirement could be <dfn>satisfied</dfn> or <dfn>further testing is needed</dfn>. 

Basically, looking to establish an "implication" relationship of "test target failure implies conformance requirement not satisfied" and "test target pass or inapplicable implies conformance requirement satisfied or inapplicable"

I was having a hard time coming up with the full description of a secondary requirement, but there were a few things I specifically thought about:
- For some test targets applicable to the rule, a failure outcome for the test target does not imply that the secondary mapping is not satisfied. If for all test targets applicable to the rule, a failure outcome *would* imply the requirement is not satisfied then this is a conformance requirement. 
- For passed outcomes for test targets, we cannot really say anything about the secondary mapping as it could be satisfied, not satisfied, or further testing needed. 
- For inapplicable outcomes, is it a requirement that the secondary requirements also be inapplicable? If we say no, then it seems like anything could be included as a secondary requirement even if it doesn't really apply.

Reply:
Just to capture some notes from our talk on 8/22. There may actually be 3 types of mappings here:
1. If the outcome is failed then the criterion is not satisfied. If the test target passes then the criterion is satisfied.
2. If the outcome is failed then the criterion is not satisfied. If the test target passes then the criterion could be satisfied or not satisfied (see color contrast example).
3. If the outcome is failed then the criterion is further testing needed. If the test target passes then the criterion is satisfied.

A few questions:
- For 1, I thought this is what was said, but I would think "further testing is necessary" might be included for a passed outcome.
- ^^ Same for 2 for passed outcomes.